apps/cookbook/src/app/examples/styling-list-example/styling-list-example.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+@Component({
+  selector: 'cookbook-styling-list-example',
+  templateUrl: './styling-list-example.component.html',
+  styleUrls: ['./styling-list-example.component.scss'],
```

Comment:
Remove reference to stylesheet
```suggestion
import { Component } from '@angular/core';
@Component({
  selector: 'cookbook-styling-list-example',
  templateUrl: './styling-list-example.component.html',
```